core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -190,6 +190,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       .orNull
     numExecutors = Option(numExecutors)
       .getOrElse(sparkProperties.get("spark.executor.instances").orNull)
+    queue = Option(queue).orElse(sparkProperties.get("spark.yarn.queue")).getOrElse("default")
```

Comment:
The default value of `spark.yarn.queue` is `defualt`, I think we should follow that. Here we just print the spark submit arguments from command line and the conf file. If the queue of change by user later, they should be aware of that.